src/client/client.cpp

```diff
@@ -121,15 +121,10 @@ Client::Client(
 
 void Client::loadMods()
 {
-	// Don't load mods twice
-	if (m_mods_loaded) {
-		return;
-	}
-
+	// Don't load mods twice.
 	// If client scripting is disabled by the client, don't load builtin or
 	// client-provided mods.
-	if (!m_modding_enabled) {
-		warningstream << "Client side scripting is disabled by client." << std::endl;
```

Comment:
instead of removing reduce noise by use infostream

Reply:
But why? It would make sense to use infostream if client-side scripting is enabled, but not when it's disabled (i.e. default setting)

Reply:
Is it useful information to log though? I'm unsure. It is the default after all. Is there a need to inform someone looking at logs that a setting is set to a default?

Reply:
This is a clientside setting, surely logging is serverside only, so logging this in infostream will be mostly useless on a client. Even if a client has a terminal open, infostream messages are not shown by default.

Reply:
I'm also unsure this info is useful, since this is the default.
Or, if clientside loggin exists and it is useful to log the state of CSM, there should be 2 messages, one for enabled, one for disabled?

I don't play on servers and have only used local servers, where debug.txt is generated by the server only, so i don't know if clients generate their own debug.txt when connected to a non-local server.

Reply:
So, i don't think shoving it into infostream is a solution, i think CSM status messaging needs some thought on what is needed and how to do it.

Reply:
This is client-side - **`client.cpp`** :)

Reply:
Testing (by joining a server on public server list (itb) with Minetest 5.1.0-dev-bb7afd3 (Linux)) also shows:
- Message shows up in terminal.
- Message is logged in `debug.txt`.

Reply:
Yes, as it should. To clarify, both server and client are capable of logging stuff, and this particular message is logged client-side.

Reply:
Thanks.

Reply:
For consistency either log all unchanged disabled-by-default features or do not log this unchanged disabled-by-default feature …